addons/mail/static/src/components/channel_member/channel_member.xml

```diff
@@ -2,23 +2,23 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChannelMember" owl="1">
-        <div class="o_ChannelMember d-flex align-items-center mx-3 my-1" t-attf-class="{{ className }}" t-att-data-partner-local-id="member.localId" t-ref="root">
-            <div class="o_ChannelMember_avatarContainer position-relative flex-shrink-0 o_cursor_pointer" t-on-click="() => channel.onClickMemberAvatar(member)">
-                <img class="o_ChannelMember_avatar rounded-circle w-100 h-100" t-attf-src="/mail/channel/{{ channel.id }}/partner/{{ member.id }}/avatar_128" alt="Avatar"/>
+        <div class="o_ChannelMember d-flex align-items-center mx-3 my-1" t-attf-class="{{ className }}" t-att-data-partner-local-id="channelMember.partner.localId" t-ref="root">
```

Comment:
Note that `ChannelMember/partner` is not necessarily defined, as you can have `ChannelMember` being a guest

Reply:
Yes, but this is for an other PR

Reply:
Ho I get it, did not remove all the guest related code yet. It's done now.